src/libraries/Native/Unix/System.Security.Cryptography.Native/apibridge.c

```diff
@@ -19,7 +19,7 @@
 
 #define CRYPTO_LOCK_X509 3
 #define CRYPTO_LOCK_EVP_PKEY 10
-
+#define CRYPTO_LOCK_BIO 21
```

Comment:
```suggestion
#define CRYPTO_LOCK_BIO 21

```

Please leave the separation between the crypto lock IDs and the SSL_CTRL IDs.